nuxeo-features/nuxeo-automation/nuxeo-automation-test/src/test/java/org/nuxeo/ecm/automation/server/test/EmbeddedAutomationClientTest.java

```diff
@@ -1001,22 +1005,54 @@ public void shouldReadContentEnricher() throws IOException {
     }
 
     /**
+     * @since 8.10
+     */
+    @Test
+    public void cannotSendCalendarParametersIfUserNotFound() throws IOException {
+        Document root = (Document) super.session.newRequest(FetchDocument.ID).set("value", "/").execute();
+        OperationRequest request = session.newRequest(AddPermission.ID);
+        GregorianCalendar begin = new GregorianCalendar(2015, Calendar.JUNE, 20, 12, 34, 56);
+        GregorianCalendar end = new GregorianCalendar(2015, Calendar.JULY, 14, 12, 34, 56);
+        try {
+            request.setInput(root)
+                   .set("username", "nonExistentMembers")
+                   .set("permission", "Write")
+                   .set("begin", begin)
+                   .set("end", end)
+                   .execute();
+        } catch (RemoteException e) {
+            String expectedMsg = "Failed to invoke operation: Document.AddPermission";
+            assertEquals(e.getMessage(), expectedMsg, e.getMessage());
+        }
+    }
+
+    /**
      * @since 7.4
      */
     @Test
     public void canSendCalendarParameters() throws IOException {
+        // Setup
+        DocumentModel testUser = userManager.getBareUserModel();
+        testUser.setProperty("user", "username", "existingMembers");
+        testUser = userManager.createUser(testUser);
+        assertNotNull(testUser.getId());
+        txFeature.nextTransaction();
+
         Document root = (Document) super.session.newRequest(FetchDocument.ID).set("value", "/").execute();
         OperationRequest request = session.newRequest(AddPermission.ID);
         GregorianCalendar begin = new GregorianCalendar(2015, Calendar.JUNE, 20, 12, 34, 56);
         GregorianCalendar end = new GregorianCalendar(2015, Calendar.JULY, 14, 12, 34, 56);
         request.setInput(root)
-               .set("username", "members")
+               .set("username", "existingMembers")
                .set("permission", "Write")
                .set("begin", begin)
                .set("end", end)
                .execute();
         // TODO NXP-17232 to use context parameters in json payload response with automation and automation client.
         // Once NXP-17232 resolved: assertions possible to get related doc ACLs.
+
+        // Tear down
+        userManager.deleteUser(testUser.getId());
```

Comment:
Please, do the tear down in a `try {} finally {}` to ensure that the user is always deleted.